homeassistant/components/history/__init__.py

```diff
@@ -211,6 +212,9 @@ def states_to_json(
     axis correctly.
     """
     result = defaultdict(list)
+    if isinstance(entity_ids, Iterable):
```

Comment:
This is not a right check, a string is iterable too.

Reply:
I guess it works because we don't pass a list. We should just guard against it being `None`

Reply:
*Gasp! Stack overflow lied to me!

Thanks for fixing.
Will `entity_ids` NEVER be a single entity_id in a string?

Reply:
based on the code, no.